src/server.rs

```diff
@@ -1121,17 +1127,12 @@ impl WebThingServer {
         });
 
         let responder = libmdns::Responder::new().unwrap();
+        self.dns_service =
+            Some(responder.register(SERVICE_TYPE.to_owned(), name.clone(), port, &["path=/"]));
 
         #[cfg(feature = "ssl")]
         match self.ssl_options {
             Some(ref o) => {
-                responder.register(
-                    "_webthing._tcp".to_owned(),
-                    name.clone(),
-                    port,
-                    &["path=/", "tls=1"],
```

Comment:
The difference here that is now omitted is the `tls=1` key/value pair.

Reply:
I missed that, apologies! I wondered why the line was duplicated ;-)

Fixed: https://github.com/WebThingsIO/webthing-rust/pull/54/commits/1a8c3170cf3dd71ac2560a57376801ce4309b966